eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```
@@ -110,6 +110,8 @@ services:
             $authorizationChecker: '@security.authorization_checker'
             $locationProvider: '@ezpublish.content_preview.location_provider'
             $controllerChecker: '@ezpublish.view.custom_location_controller_checker'
+            $debugMode: '%kernel.debug%'
+            $logger: '@logger'
         tags:
             - { name: controller.service_arguments }
```

Nowadays we try to send logs to specific monolog channels. To do so please add:
```suggestion
            - { name: controller.service_arguments }
            - { name: monolog.logger, channel: ibexa.controller }
```
note that `channel` is arbitrary / gut feeling choice. I briefly discussed this with @Steveb-p  and we've chosen here `ibexa.controller`.